### DISCUSSION TITLE
[API Request] Add Starred/favorite property to Playlists as well

### DISCUSSION THREAD

**Discussion by dweymouth** [opensubsonic/open-subsonic-api#43] [API requests]
### Type of change

API extension

### Proposal description

Currently, users can star/favorite artists, albums, and songs. We should add the ability to star/favorite playlists as well. 

Context: https://www.reddit.com/r/navidrome/comments/15055m8/favoriting_playlists/

### Backward compatibility

- [ ] No backward compatibility impact.

### API details

Add `starred` property to playlists, and ability to star playlists, just like albums, artists, and songs

**Comment by lachlan-00** [comment]
anything with an id should be able to be rated and starred so this is fine with me
